index.html

```diff
@@ -231,6 +230,33 @@ <h3>Languages</h3>
         <img src="assets/vector.svg " alt="">
       </div>
   </section>
+
+  <div class="container-white">
+ 
+    <section class="contact-me">
+      
+      <form action="https://formspree.io/f/xbjevbkv" method="POST">
+        <div class="background">
+        <h2>contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+          I’d love to help with it!</p>
+        <label for="name">
+          <input type="text" id="name" placeholder="Lorem Ipsum" required maxlength="30">
+        </label>
+        <label for="email">
+          <input type="email" id="message" placeholder="loremipsum@gmail.com" required/>
+        </label>
+        <label for="message-box">
+          <textarea id="message-box" placeholder="Please enter your message here" required maxlength="500"></textarea>
+        </label>
+        <button class="el-botones">
+          <p>Get in touch</p>
+        </button>
+        </div>
+      </form>
```

Comment:
- [ ] Please I highly suggest adding `name` attribute to all your input elements and `type="submit"` to your `button` for the contact form functionality to be implemented. 

Example: 

```
<form action="https://formspree.io/f/xbjevbkv" method="POST">
  <div class="background">
    <h2>contact me</h2>
    <p>If you have an application you are interested in developing, a feature that you need built, or a project that needs coding. I’d love to help with it!</p>
    <label for="name">
      <input type="text" id="name" name="name" placeholder="Lorem Ipsum" required maxlength="30">
    </label>
    <label for="email">
      <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required>
    </label>
    <label for="message-box">
      <textarea id="message-box" name="message" placeholder="Please enter your message here" required maxlength="500"></textarea>
    </label>
    <button type="submit" class="el-botones">
      <p>Get in touch</p>
    </button>
  </div>
</form>
```

_**This is a screenshot of the response after the Get in touch button has been clicked.**_

![Screenshot (342)](https://github.com/santitoot/finish-mobile-version/assets/78104544/6a36eeee-e056-4499-aa11-81388460f5b1)

Reply:
- [ ] Kindly implement this change, since your form currently isn't working as intended 🧑‍🏭